etcdserver/api/v3rpc/maintenance.go

```diff
@@ -38,6 +39,7 @@ type BackendGetter interface {
 }
 
 type Alarmer interface {
+	Alarms() []*pb.AlarmMember
```

Comment:
can we document this public interface? I know there are existing undocumented methods, but we do need to improve the codebase in general :)

Reply:
Yep. This is an existing interface that I just added to this Alarmer interface. Will document it.